shared/Workflow.ts

```diff
@@ -36,41 +34,18 @@ export class Workflow {
       return this.#workflows[type]
     }
 
-    const [workflow, statuses] = await Promise.all([
-      this.#fetchWorkflow({ type, accessToken }),
+    const [statuses] = await Promise.all([
+      // this.#fetchWorkflow({ type, accessToken }),
       this.#fetchStatuses({ type, accessToken })
     ])
```

Comment:
Might as well remove this?
```suggestion
    const statuses = await this.#fetchStatuses({ type, accessToken })
```